doc/newsfragments/2528_new.export_context.rst

```diff
@@ -0,0 +1 @@
+Introduced new :py:class:`ExportContext <testplan.common.exporters.ExportContext>` class for exporters to communicate with each other.
```

Comment:
As suggested below, probably breaking change, so needs to be reflected.

Reply:
It shouldn't be breaking. The case when the user uses a custom exporter which does not implement the new features is handled and will show a warning message for now. (This will be removed in the future). Let me test it more.